Slime Mall/Assets/Scripts/Managers/GameManager.cs

```diff
@@ -16,10 +16,8 @@ public class GameManager : MonoBehaviour
     [Range(0, 360)][SerializeField][Tooltip("Length of initial game session")] float SessionLength = 120;
     [Tooltip("Score of current session")]  int Score = 0;
     [Tooltip("")] int HighScore = 0;
-    [Tooltip("How many points you gain per kill")] int KillScoreValue = 10;//SUGGESTION: Move it to the NPC's in case we have varying NPC's that offer different score
     [Tooltip("How much time you gain per kill")] int KillTimeValue = 1;//SUGGESTION: Move it to the NPC's in case we have varying NPC's that offer different time gain
```

Comment:
Think this is a good idea to future proof the code by having this inside the NPC class